manage.py

```diff
@@ -2,10 +2,14 @@
 import os
 import sys
 
+from opentelemetry.instrumentation.django import DjangoInstrumentor
+
 if __name__ == "__main__":
     os.environ.setdefault("DDM_CONTAINER_NAME", "app")
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "usaspending_api.settings")
 
+    DjangoInstrumentor().instrument()
```

Comment:
If I am following this correctly, this DjangoInstrumentor should create additional logging that Django normally outputs, but we aren't currently capturing?